postgres-appliance/scripts/post_init.sh

```diff
@@ -158,6 +158,7 @@ CREATE EXTENSION IF NOT EXISTS set_user SCHEMA public;
 ALTER EXTENSION set_user UPDATE;
 GRANT EXECUTE ON FUNCTION public.set_user(text) TO admin;
 GRANT EXECUTE ON FUNCTION public.pg_stat_statements_reset($RESET_ARGS) TO admin;"
+    if [ "$PGVER" -lt 10 ]; then echo "GRANT EXECUTE ON FUNCTION pg_catalog.pg_switch_xlog() TO admin;"; else echo "GRANT EXECUTE ON FUNCTION pg_catalog.pg_switch_wal() TO admin;"; fi;
```

Comment:
Lets better format it:
```suggestion
    if [ "$PGVER" -lt 10 ]; then
        echo "GRANT EXECUTE ON FUNCTION pg_catalog.pg_switch_xlog() TO admin;"
    else
        echo "GRANT EXECUTE ON FUNCTION pg_catalog.pg_switch_wal() TO admin;"
    fi
```